src/main/resources/theme/play/custom_games.fxml

```diff
@@ -86,7 +87,7 @@
                     </content>
                 </ScrollPane>
                 <AnchorPane fx:id="gameViewContainer" maxHeight="1.7976931348623157E308"
-                            maxWidth="1.7976931348623157E308" GridPane.rowIndex="1" prefWidth="10000"/>
+                            maxWidth="1.7976931348623157E308" prefWidth="10000" GridPane.rowIndex="1"/>
```

Comment:
Maybe thi s causes issue tho i did not set this this was scene builder...

Reply:
I don't see the difference, am i blind? Do you mean the order?

Reply:
oh no i thought it added prefWidth